examples/todo/todo.py

```diff
@@ -134,9 +151,10 @@ def create_app(name):
     app = Flask(name)
 
     authenticator = HeaderApiKeyAuthenticator(header='X-MyApp-Key')
-    # The HeaderApiKeyAuthenticator does super simple authentication, designed for
-    # service-to-service authentication inside of a protected network, by looking for a
-    # shared secret in the specified header. Here we define what that shared secret is.
+    # The HeaderApiKeyAuthenticator does super simple authentication, designed 
+    # for service-to-service authentication inside of a protected network, by 
+    # looking for a shared secret in the specified header. Here we define what 
+    # that shared secret is.
```

Comment:
(ditto about bumping to 88)

Reply:
This becomes:
`    # The HeaderApiKeyAuthenticator does super simple authentication, designed for
    # service-to-service authentication inside of a protected network, by looking for a
    # shared secret in the specified header. Here we define what that shared secret is.
`
note: I will not add this change here, since it is a style migration that is independent of original scope of 'add tags to the todo example'. I'm happy to put this into a new PR if that is what we decide (see #60 )